src/OctoshiftCLI.Tests/Octoshift/Services/ConfirmationServiceTests.cs

```diff
@@ -14,16 +14,17 @@ public class ConfirmationServiceTests
         private readonly string cancelationOutput;
         private readonly string confirmationOutput;
         private string _consoleOutput;
+        private int _exitOutput;
         private int numOfCalls;
         private ConsoleKey passedKey;
         #endregion
 
         #region Constructor
```

Comment:
we don't use #regions anywhere else in the code, and they are generally frowned upon in the .Net world.

Reply:
Huh! Hadn't heard that before. We used them at both my last gigs. Now I know :)
Removed in 93f291843aeaab968bb4950fdc86cef38cd8602c